nemo/collections/asr/parts/activations.py

```diff
@@ -18,10 +18,8 @@
 __all__ = ['Swish']
 
 
-class Swish(nn.Module):
+class Swish(nn.SiLU):
```

Comment:
Why not drop this class and use nn.SiLU directly?
We may replace all nemo.Swish with nn.SiLU in the code?

Reply:
In this way we may have the following for backward compatibility:
jasper_activations = {"hardtanh": nn.Hardtanh, "relu": nn.ReLU, "selu": nn.SELU, "swish": nn.SiLU, "silu": nn.SiLU}

Reply:
Swish is mathematically the same, but its non-obvious at first glance that they are the same (I for one kept searching for swish in PT and never found it until I went through all the activations and found SiLU)..